<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ETEPC_calc</class>
 <widget class="QMainWindow" name="ETEPC_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>295</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>223</width>
    <height>295</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>223</width>
    <height>295</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ETEPC-5000</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="display">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>208</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>208</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>208</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei Mono</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>161</width>
      <height>176</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="Button_plusminus">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Button_mul">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>ร</string>
       </property>
       <property name="shortcut">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="Button_div">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>รท</string>
       </property>
       <property name="shortcut">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="Button_minus">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="shortcut">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Button_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>7</string>
       </property>
       <property name="shortcut">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Button_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <property name="shortcut">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="Button_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>9</string>
       </property>
       <property name="shortcut">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" rowspan="2">
      <widget class="QPushButton" name="Button_plus">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>65</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <property name="shortcut">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="Button_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>4</string>
       </property>
       <property name="shortcut">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="Button_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="shortcut">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="Button_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>6</string>
       </property>
       <property name="shortcut">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="Button_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="shortcut">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="Button_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="shortcut">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="Button_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="shortcut">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" rowspan="2">
      <widget class="QPushButton" name="Button_result">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>65</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>65</height>
        </size>
       </property>
       <property name="text">
        <string>=</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QPushButton" name="Button_0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="shortcut">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="Button_DPoint">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>,</string>
       </property>
       <property name="shortcut">
        <string>,</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>32</width>
      <height>176</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Button_C">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>C</string>
       </property>
       <property name="shortcut">
        <string>Del</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_fact">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>n!</string>
       </property>
       <property name="shortcut">
        <string>!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_ln">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_power">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>xn_button.png</normaloff>xn_button.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Ans">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Ans</string>
       </property>
       <property name="shortcut">
        <string>Space</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>223</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="action_Help"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Button_0</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>num_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_1</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>num_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_2</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>num_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_3</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>num_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_4</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>num_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_5</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>num_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_6</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>num_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_7</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>num_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_8</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>num_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_9</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>num_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_plus</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>operator2_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_minus</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>operator2_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_mul</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>operator2_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_div</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>operator2_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_result</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>result_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_DPoint</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>dpoint_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_ln</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>operator1_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_fact</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>operator1_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_plusminus</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>neg_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_C</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>clear_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_power</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>operator2_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_About</sender>
   <signal>triggered()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>action_About()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Help</sender>
   <signal>triggered()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>action_Help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_Ans</sender>
   <signal>clicked()</signal>
   <receiver>ETEPC_calc</receiver>
   <slot>ans_hit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>num_hit()</slot>
  <slot>operator1_hit()</slot>
  <slot>operator2_hit()</slot>
  <slot>clear_hit()</slot>
  <slot>neg_hit()</slot>
  <slot>result_hit()</slot>
  <slot>action_About()</slot>
  <slot>action_Help()</slot>
  <slot>ans_hit()</slot>
  <slot>dpoint_hit()</slot>
 </slots>
</ui>
